commit 5ab35babadbd4bf5d21d461911fc2ddbfe56229d
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Wed Nov 14 11:54:46 2018 +0000

    vcsm: Fix an NULL dereference in the import_dmabuf error path
    
    resource was dereferenced even though it was NULL.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/char/broadcom/vc_sm/vmcs_sm.c b/drivers/char/broadcom/vc_sm/vmcs_sm.c
index 1bc37ee88225..56a21658b538 100644
--- a/drivers/char/broadcom/vc_sm/vmcs_sm.c
+++ b/drivers/char/broadcom/vc_sm/vmcs_sm.c
@@ -2315,8 +2315,8 @@ int vc_sm_ioctl_import_dmabuf(struct sm_priv_data_t *private,
 	return 0;
 
 error:
-	resource->res_stats[IMPORT_FAIL]++;
 	if (resource) {
+		resource->res_stats[IMPORT_FAIL]++;
 		vc_sm_resource_deceased(resource, 1);
 		kfree(resource);
 	}
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1352_5ab35babadbd4bf5d21d461911fc2ddbfe56229d.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1352_5ab35babadbd4bf5d21d461911fc2ddbfe56229d.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
